html/changelogs/ParadoxSpace - squidbillies.yml

```diff
@@ -0,0 +1,45 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#################################
+
+# Your name.  
+author: ParadoxSpace
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
```

Comment:
This is a ridiculous, Nanako-level changelog, Jesus crhist.
rscadd "Adds empath reading to Skrell, allowing one to read the intent and thoughts of another"
rscadd "Adds shortwave telepathy and commune with creature (possibly an explanation of what this actually does) to Skrell"
rscadds "Adds a set thoughts verb, allowing one to manually set what they're thinking about."

The technical shit doesn't belong in a changelog, and tweaks to an unpublished PR don't need to be recorded in a changelog. Nobody gives a shit about the chaplain and his 5% chance, they can find that out on the wiki or some shit.

Otherwise, this implementation is pretty clumsy in regards to the set thought verb, and I am pretty sure we were trying to avoid explicit language telepathy with Skrell.

Reply:
Cool, it's a good thing none of you were working on it after it being promised two years ago. I'd say pump it out, get it done and improve upon it later.

Reply:
That's a hard no from me, sarge.

Reply:
you can't blame coders for lore that's a pain in the ass to represent via game mechanics Xander.

Any actual *good* implementation of mind reading would be massively broken regardless. but a "what're you thinking" verb adds less than pose, and will almost never be used.


Any thoughts system should be automated tbh, potentially based on recent actions/attacks?